labcore/instruments/qick/qick_sweep_v2.py

```diff
@@ -110,6 +110,13 @@ def collect(self, *args, **kwargs):
                 return_data[ds.name] = np.arange(cfg['steps'])
                 sweepIdx += 1
 
+<<<<<<< HEAD
+=======
+        # Reformat the independent variables
+        # Independent (sweep) variables have to be reformatted such that xArray and plottr can
+        # correctly recognize the axis being swept. Without reformatting the swept variable,
+        # the program won't be able to correctly set up the axis being swept.
+>>>>>>> 2132e06 (Removed qick_sweep v1)
```

Comment:
These look like a merge conflict git message, I think if we run this python will not be able to use this file.